src/services/domain/membership/membership.dto.user.result.entry.ecoverse.ts

```diff
@@ -23,13 +24,24 @@ export class MembershipUserResultEntryEcoverse extends MembershipResultEntry {
   })
   userGroups: MembershipResultEntry[] = [];
 
+
+  @Field(() => MembershipCommunityResultEntry, {
+    description: 'The community that represents this ecoverse',
+  })
+  community: MembershipCommunityResultEntry;
```

Comment:
Do not think this is needed also in the ecoverse entry given there is the top level communities array.